src/llm/BUILD

```diff
@@ -93,8 +93,10 @@ cc_library(
 cc_library(
     name = "genai_servables",
     hdrs = ["servable.hpp", "servable_initializer.hpp", 
-            "continuous_batching/servable.hpp", "continuous_batching/llm_executor.hpp", "continuous_batching/servable_initializer.hpp"],
-    srcs = ["servable.cpp", "servable_initializer.cpp", "continuous_batching/servable.cpp", "continuous_batching/servable_initializer.cpp"],
+            "continuous_batching/servable.hpp", "continuous_batching/llm_executor.hpp", "continuous_batching/servable_initializer.hpp",

```

Comment:
why do we put everything to one general target?
Adrian is working on modularization - this is subject to that, since multiple implementations will have different dependnecies, maybe split it to different targets?